### ISSUE TITLE
VicReg implementation gives "nan" loss

### ISSUE THREAD

**Issue by ManafMukred** [tensorflow/similarity#335]
I was trying to explore other algorithms like vicreg using LAMB & LARS optimizer, but in both cases the loss is "nan"

`Epoch 1/200
175/175 - 49s - loss: nan - proj_std: nan - val_loss: nan - val_proj_std: nan - binary_accuracy: 0.1000 - 49s/epoch - 281ms/step
Epoch 2/200
175/175 - 36s - loss: nan - proj_std: nan - val_loss: nan - val_proj_std: nan - binary_accuracy: 0.1000 - 36s/epoch - 203ms/step
Epoch 3/200
175/175 - 36s - loss: nan - proj_std: nan - val_loss: nan - val_proj_std: nan - binary_accuracy: 0.1000 - 36s/epoch - 204ms/step
Epoch 4/200`

any suggestions?

**Comment by owenvallis** [comment]
Hi @ManafMukred, can you share some more details about the model architecture, hyperparameters, and dataset?

**Comment by ManafMukred** [comment]
@owenvallis I used the same notebook [`here `](https://github.com/tensorflow/similarity/blob/master/examples/unsupervised_hello_world.ipynb) , and I tried to use the LARS optimizer also with lr = 0.2 * int(BATCH_SIZE / 256) same as the paper says. 
but I get the same error

**Comment by owenvallis** [comment]
Thanks for the details, I'll take a look and see if I can repro the issue on my side.
